### DISCUSSION TITLE
Could we start a Discord? Migrating from v5 to v6 might benefit from an active community

### DISCUSSION THREAD

**Discussion by clmnin** [fabricjs/fabric.js#10244] [Polls]
I think it would be beneficial to have a community where we can ask questions about Fabric.js—not just limited to the v5 to v6 migration, but for other topics as well. We don’t really know who else is using Fabric.js, if there are freelance opportunities, and so on

**Comment by asturur** [comment]
Discord would only decentralize the information in two places imho.
Also who would be answering in this Discord?
I do not think Discord is a great way for knowledge sharing that is not based on the immediate ' i need answer for problem X and i will ask it to someone '

**Comment by asturur** [comment]
Of course anyone can do it.
If someone feels he wants to run a FabricJS discord can open it and run it. Is not gatekept by me.

**Comment by clmnin** [comment]
Ok. If the maintainers don't want to run a discord then I'll close this.

The current problem is that you or another maintainer are the only ones answering questions on [github discussions](https://github.com/fabricjs/fabric.js/discussions). Which can take a while. And since there is no documentation on migration from v5 to v6 (the closest thing is this #8299), a place to ask a question and get a quick response would have been great.

**Comment by asturur** [comment]
The only maintainer answering questions is mostly me, and i should spend time writing documentation and not answering ad hoc problems of particular apps.
Time is few lately
